docs/reference/patterns/distributed-locks.md

```diff
@@ -206,6 +206,20 @@ However this does not technically change the algorithm, so the maximum number
 of lock reacquisition attempts should be limited, otherwise one of the liveness
 properties is violated.
 
+### Disclaimer about consistency
+
+Please consider thoroughly reviewing the [Analysis of Redlock](#analysis-of-redlock) section at the end of this page.
+Martin Kleppman's article and antirez's answer to it are very relevant.
+If you are concerned about consistency and correctness, you should pay attention to the following topics:
+
+1. You should implement fencing tokens.
+  This is especially important for processes that can take significant time and applies to any distributed locking system.
+  Extending locks' lifetime is also an option, but don´t assume that a lock is retained as long as the process that had acquired it is alive.
+2. ["Redis is not using monotonic clock for TTL expiration mechanism and there are no plans to use them
+for now" (August, 2022)](https://github.com/redis/redis/pull/7644#issuecomment-1206815232).
```

Comment:
@itamarhaber don't you think this part is excessive?
```suggestion
2. Redis is not using monotonic clock for TTL expiration mechanism.
```

Reply:
@itamarhaber @oranagra the reason for that "for now" and the link is, from may POV, a try to given some more context to a reader that might be wondering is the monotonic clock topic is something that eventually could happen. Basically the path I followed to that issue and this PR.
If you guys consider this too much I'm fine applying @oranagra suggestion.
I have very limited access to my laptop this week, though 😅

Reply:
I think that generally the docs specify how things work, and not how they could have worked and why it's not that way, what was planned or decided against...

@itamarhaber can make that decision and apply / edit the text if needed.

Reply:
I admired @jhmilan research capabilities in finding the discussion, but on 2nd thought I tend to agree with @oranagra - sometimes less is better.